class.jetpack.php

```diff
@@ -4658,6 +4652,12 @@ public static function build_authorize_url( $redirect = false, $iframe = false )
 
 		$connection = self::connection();
 
+		$calypso_env = self::get_calypso_env();
```

Comment:
Where this lived previously, we'd add the `calypso_env` arg for register URLs too. After the change, this will no longer be the case. Is this something we should be worried about?

Reply:
Nice catch. Addressed in 20e803e80e3959912ccdcef5c80982fd6d781537